File: pep/level1/basics/gettingstarted/BasePower.java
import java.util.Scanner;

public class BasePower {

  public static void main(String [] args) {

    int base, power, accum = 1;
    Scanner s = new Scanner(System.in);
    System.out.println("Enter a base:");
    base = s.nextInt();
    System.out.println("Enter a power:");
    power = s.nextInt();

    while(power > 0) {
      accum = base * accum;
      power = power - 1;
    }

    System.out.println(accum);
  }
}

File: pep/level1/basics/gettingstarted/BasePower.cpp
#include <iostream>
using namespace std;

int main() {

  int base;
  cout << "Enter a base \n";
  cin >> base;

  int pow;
  cout << "Enter a power \n";
  cin >> pow;

  int acc = 1;

  while(pow > 0) {
    acc = base * acc;
    pow -= 1;
  }
  cout << acc;

  return 0;
}
